src/GitHub/Data/Request.hs

```diff
@@ -174,7 +175,7 @@ type Request = GenRequest 'MtJSON
 query :: Paths -> QueryString -> Request mt a
 query ps qs = Query ps qs
 
-pagedQuery :: FromJSON a => Paths -> QueryString -> FetchCount -> Request mt (Vector a)
+pagedQuery :: FromJSON a => Paths -> QueryString -> FetchCount -> GenRequest mt rw (Vector a)
```

Comment:
revert'. Use `PagedQuery` where more relaxed version is needed.

Reply:
Will do